health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -895,26 +909,99 @@
+        
+            return existingColumns.every((existingItem, index) => {
+              const newItem = newColumnValues[index];
+              return existingItem?.value === newItem?.value;
+            });
+          };
+        
+          // Skip updatePlan if both sets of columns are the same
+          if (areColumnsSame()) {
+            setCurrentKey(prev => prev + 1);
+            setCurrentStep(prev => prev + 1);
+            Digit.Utils.microplanv1.updateUrlParams({ isLastVerticalStep: null });
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)